src/arrays/what-is-practicum.js

```diff
@@ -1,60 +1,80 @@
+import image from '../images/practicum-logo.png';
+
 const practicum = [
   {
     id: 1,
     owner: 'practicum',
     message:
       'Practicum by Yandex is a kind of boot camp. Our goal is to help ambitious and passionate people to master it and find their first job. At the same time, the level of our training allows graduates to carry out high-quality real projects of companies under the review of experienced mentors.',
     direction: 'row',
-    'background-color': 'orange',
+    backgroundColor: 'orange',
     text: 'white',
-    'bacground-image': 'image',
+    backgroundImage: image,
+    displayLogo: 'block',
+    borderRadius: '14px 14px 14px 0px',
```

Comment:
See comment in stylesheet about using pseduoclasses as an alternative. Again, definitely not necessary! This works and makes sense givin the logic, just another method.

Reply:
I could clean this up by using ```css :nth-child(even) ``` but I was already invested in doing it this way. I'll put it as a refactor note.